device/arista/x86_64-arista_7800r3a_36d2_lc/Arista-7800R3A-36D2-C36/0/buffers_defaults_t2.j2

```diff
@@ -0,0 +1,37 @@
+{%- set default_cable = '300m' %}
+
+{%- macro generate_port_lists(PORT_ALL) %}
+    {# Generate list of ports #}
+    {%- for port_idx in range(0,144,8) %}
+        {%- if PORT_ALL.append("Ethernet%d" % (port_idx)) %}{%- endif %}
+    {%- endfor %}
+{%- endmacro %}
+
+{%- macro generate_buffer_pool_and_profiles() %}
+    "BUFFER_POOL": {
+        "ingress_lossless_pool": {
+            "size": "6441610000",
+            "type": "both",
+            "mode": "dynamic",
+            "xoff": "11354112"

```

Comment:
Rest of the setting look ok. Can you please share calculation details for XOFF ? Also, are these profiles going to cover all Wolverine SKUs ?

Reply:
Hi @kenneth-arista , @ysmanman - reminder of this

Reply:
As per discussion, these settings are to unblock testing and not the final one. I will raise another PR to update these settings based on cable length we are planning to use.